error.vue

```diff
@@ -8,27 +8,16 @@ const { error } = defineProps<{
 
 // add more custom status codes messages here
 const errorCodes: Record<number, string> = {
-  404: 'Page not found',
+  404: 'Sorry, we can\'t find that page',
+  500: 'Sorry, something went wrong',
 }
 
 if (process.dev)
   console.error(error)
 
 const defaultMessage = 'Something went wrong'
```

Comment:
I know this wasn't localized in the previous code. But might be worthwhile to wrap it here... unless something is preventing the error pages from loading localization.

Reply:
Localized!